SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslationManager/SpatialOSDispatcherMock.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
This is currently in the TranslationManager's testing folder, but I think we should make it a generic mock that all tests can use, and then when someone implements a test of a class that needs (for example) OnAddEntity, then they would fill in the implementation below for that method. It would be a way that adding testing functionality would be more incremental.